bin/plot_coverage.py

```diff
@@ -131,7 +135,11 @@ def plot_coverage(depths: pd.DataFrame, resistance_genes: dict={}, sample_name:
         plt.title(
             f"Percent bases with coverage above {threshold}X: {percent_coverage_above_threshold: .1f}% | Rolling window: {rolling_window} nt"
         )
-        plt.suptitle(f"Ref: {depths.iloc[0].chrom} | Sample: {sample_name}")
+        try:
+            plt.suptitle(f"Ref: {depths.iloc[0].chrom} | Sample: {sample_name}")
+        except IndexError:
+            plt.suptitle(f"Ref: H37Rv | Sample: {sample_name}")
```

Comment:
I don't know if we want to put H37Rv here or the cause of not having the chromosome info

Reply:
Hmmm... I think either is fine. By default tbprofiler uses `Chromosome` as the contig ID which is a bit annoying. I think `H37Rv` is fine.